### PR TITLE
Add jupyterhub runbook to users tab

### PR DESCRIPTION
**Pull request by Shreyanand**
Add source and link for SRE repo jupyterhub runbook to users tab on the website.

### CODE REVIEW

**Comment by Shreyanand** [comment]
cc @4n4nd

**Review by oindrillac — COMMENTED** [review]
/lgtm

File: config/users.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@
   href: /users/support/
 - label: Jupyterhub
   links:
+    - label: JupyterHub runbook for troubleshooting
```
Comment (4n4nd) [inline config/users.yaml]: I think we can just change this to `Troubleshooting`

Comment (Shreyanand) [reply config/users.yaml]: No troubles :)

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**